validation-test/stdlib/FixedPointConversion.swift.gyb

```diff
@@ -97,6 +124,105 @@ FixedPointConversionFailure.test("${Other}To${Self}Conversion/dest=${testValue}"
 %     end # for testValue in ...
 %   end # for in all_integer_types (Other)
 % end # for in all_integer_types (Self)
+
+%   for Other, otherSignificandBits in floatNameToSignificandBits.iteritems():
```

Comment:
Yes, variables stay around, and yes, gyb is more permissive than we would like it to be :(